controllers/drclusters.go

```diff
@@ -120,7 +120,7 @@ func objectsToDeploy(hubOperatorRamenConfig *rmn.RamenConfig) ([]interface{}, er
 	ramenConfig := &drClusterOperatorRamenConfig
 	drClusterOperatorNamespaceName := drClusterOperatorNamespaceNameOrDefault(ramenConfig)
 	ramenConfig.LeaderElection.ResourceName = drClusterLeaderElectionResourceName
-	ramenConfig.RamenControllerType = rmn.DRCluster
+	ramenConfig.RamenControllerType = rmn.DRClusterType
```

Comment:
In addition to renaming this variable, it would also be good to rename this file to avoid confusion.

Reply:
later once refactor is complete,  but yes possibly split into 2 files drpolicy and drcluster.

Reply:
Done